build.sbt

```diff
@@ -67,4 +71,11 @@ lazy val idGenerator = (project in file("idGenerator"))
   .settings(
     version := "1.0.0",
     name := "id-generator",
-    libraryDependencies += scalaLogging)
\ No newline at end of file
+    libraryDependencies ++= scalaLogging)
```

Comment:
Shouldn't there be either dep on `scala-loggin_2.11` or the `-api_2.10` variant depending on scala version?